ledger-api/grpc-definitions/com/daml/ledger/api/v1/version_service.proto

```diff
@@ -13,6 +13,8 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 service VersionService {
 
   // Read the Ledger API version
+  // Errors:
+  // - ``INTERNAL``: if the internal node configuration is broken
```

Comment:
Not sure this is very useful. In general, any time a user sees an error with status code `INTERNAL`, I would rather consider it a bug than a documented feature.

Reply:
Yes, you're right. I'll remove this.
However, this makes me think if we should document "default" behaviour in case of errors (if there is any), like returning `UNKNOWN`. Is there a default error mechanism for gRPC codes?

Reply:
Fixed.